types/microvm/microvm.go

```diff
@@ -30,15 +30,15 @@ type VMSpec struct {
 	MemoryMb int64 `json:"memoryMb"`
 
 	// RootVolume specifies the volume to use for the root of the microvm.
-	// +kubebuilder:validation:Required
+	// +optional
```

Comment:
Hmmm, based on the attached issue I didn't expect this change and the one below. Is that intentional? And don't you have to rebuild something to generate some manifests somewhere? :D

Reply:
Yes. Not confusing at all. :D

Reply:
> I didn't expect this change and the one below. Is that intentional?

Actually yeh this might be a mistake, it's been ages since I did this 🤣

Reply:
Actually no they are correct. These used to be required, now we supply defaults if they empty

Reply:
Okay, nice. :)